### PR TITLE
Password as only spaces bug.

### PR DESCRIPTION
**Pull request by MikeProduce**
added custom validation rule to SignupSchema in the useRegisterManager hook that ensures that the password field does not contain only spaces.

Also just a note. This is my first time contributing to any kind of open source. Any feedback is appreciated.
thanks guys!

### CODE REVIEW

File: src/features/authorization/managers/registerManager.ts [hunk 1]
```diff
@@ -24,10 +24,13 @@ export const useRegisterManager = () => {
...
       .required(t('signup:required')),
-    password: Yup.string()
-      .min(8, t('signup:passwordLengthMin'))
-      .max(64, t('signup:passwordLengthMax'))
-      .required(t('signup:required')),
+    password: Yup.string()ß
+      .min(8, 'Password must be at least 8 characters long')
+      .max(64, 'Password can be up to 64 characters long')
+      .required('Required field')
+      .test('no-spaces', 'Password cannot contain only spaces', (value) => {
```
Comment (Ryczko) [inline src/features/authorization/managers/registerManager.ts:31]: We should block not also passwords with only spaces, but also any space in password. We should use regex expression here that will allow only specific signs.

File: src/features/authorization/managers/registerManager.ts [hunk 2]
```diff
@@ -24,10 +24,13 @@ export const useRegisterManager = () => {
     email: Yup.string()
       .email(t('signup:incorrectEmail'))
       .required(t('signup:required')),
-    password: Yup.string()
-      .min(8, t('signup:passwordLengthMin'))
-      .max(64, t('signup:passwordLengthMax'))
-      .required(t('signup:required')),
+    password: Yup.string()ß
+      .min(8, 'Password must be at least 8 characters long')
```
Comment (Ryczko) [inline src/features/authorization/managers/registerManager.ts:28]: All validation messages should be taken from translation file.

**Review by Ryczko — CHANGES_REQUESTED** [review]
Hello, thanks for contribution! Please look on some feedback :)

File: src/features/authorization/managers/registerManager.ts (hunk 2 above)
Comment (MikeProduce) [reply src/features/authorization/managers/registerManager.ts:28]: Awesome will go back and fix this ! Thanks for the feedback !

File: src/features/authorization/managers/registerManager.ts (hunk 1 above)
Comment (MikeProduce) [reply src/features/authorization/managers/registerManager.ts:31]: Sounds good will take a look later today thanks !

**Pull request closed without merging**